src/sentry/search/events/fields.py

```diff
@@ -2544,7 +2544,13 @@ def __init__(
                             # in the json query syntax.
                             # TODO(snql-migration): Once the trends endpoint is using snql, we should update it
                             # and flip these conditions back
-                            Function(args["condition"], [args["middle"], self.column("timestamp")]),
+                            Function(
+                                args["condition"],
+                                [
+                                    Function("toDateTime", [args["middle"]]),
```

Comment:
This additional casting isn't necessary, clickhouse will handle the string just fine, but adding this for safety